astronomy/April-2022-astronomy-minutes.md

```diff
@@ -0,0 +1,80 @@
+# April 2022 Meeting of the Astronomy Curriculum Advisors  
+
+There were initially two meetings arranged. But at the end, only one meeting took place on **March 25, 2022, 16.00-17.00 CET**.
+
+
+## Attendants
+
+* Azalee Bostroem
+* Heather Andrews
+* Ivelina Momcheva
+* Phil Rosenfield
+* Adam Hughes
+* Rachel Lee McClure
+* Erin Becker
+
+## Roles
+
+**Facilitator:** Rachel
+
+**Notetaker:** Phil
+
+**Timekeeper:** Erin
+
+## Agenda
+
+### Introductions
+
+7 very interesting and excellent people!
+
+### Discussion on transition of curriculum to Stable
+
+* Curriculum has gone through the following phases: **dev**, **alpha**, **beta**. It has been taught several times, now (proposed) ready to transition to stable.
+
+* **First time a CAC is making this decision** (rather than core carpentries team).
+
+* **First new curriculum since 2018**. 
+
+*  **Can this be taught over 2 days?**
+	- Timing was set by The Carpentries.
+	- All testing was done virtually (pandemic times), there are notes on what to cut for time or other reasons.
+	- Still need more data points as we transition to in-person and more people teach it.
+
+*  **What is the recommended level of participants?**
+	- It's not immediately clear that this curriculum is intermediate / advanced. It has prerequisites (it is "level 2").
+
+*  **Any major gaps (beyond the issues raised by previous workshop feedback)?**
+	- No major gaps.
+	- (Azalee) We had funding to hire a curriculum developer to do the major lift; since then it's been moving topics around, simplifying whether to write a function or not. Most current issues will be closed by current pull requests; others are flagged by what needs to be done before release.
+	- (Azalee) Worried that we don't have any data points on in-person. But don't think it's important to keep this in beta until we go in-person. Don't think anything will be a big enough change. [Consensus agreement]
+
+*  **Other comments?**
```

Comment:
I included this in the discussion above - I think it makes sense to include it in context